_translator-files/po/sv-SE/1-index.po

```diff
@@ -81,33 +98,35 @@ msgid ""
 "Check out the [documentation](wiki/Getting-Started) and consider the [troubleshooting section](wiki/Client-Troubleshooting)!\n"
 "You can also ask on the [forums](https://github.com/jamulussoftware/jamulus/discussions).\n"
 msgstr ""
+"Kolla in [dokumentationen](wiki/Getting-Started) och överväg att läsa avsnittet om [troubleshooting section](wiki/Client-Troubleshooting)! \n"
+"Du kan också ställa frågor på [forums](https://github.com/jamulussoftware/jamulus/discussions).\n"
```

Comment:
Not sure if "forums" is correct in Swedish.

Reply:
Text in square brackets needs translating (`[troubleshooting section]`), too.